jobsub/examples/pALPIDEfs/steering-templates_pALPIDEfs_7/hotpixel-tmp.xml

```diff
@@ -26,7 +26,7 @@
    <global>
       <parameter name="LCIOInputFiles"> @LcioPath@/@FilePrefix@-converter.slcio </parameter>
       <parameter name="GearXMLFile" value="@GearFilePath@/@GearFile@"/>
-      <parameter name="MaxRecordNumber" value="@nEventsForHotpixel@"/>
+      <parameter name="MaxRecordNumber" value="@MaxRecordNumber@"/>
```

Comment:
Where do you set MaxRecordNumber? I think this value could go into the config file.